pixeltable/catalog/catalog.py

```diff
@@ -200,6 +202,7 @@ def __init__(self) -> None:
         self._in_write_xact = False
         self._x_locked_tbl_ids = set()
         self._modified_tvs = set()
+        self._current_tx_undo_actions = []
```

Comment:
'xact' is used elsewhere in the code to mean 'transaction' (following common practice in the dbms community), let's stick with that.

Reply:
Changed to `_undo_actions`.